src/openforms/js/components/form/edit/date-edit-form.js

```diff
@@ -62,34 +63,31 @@ const getMinMaxValidationEditForm = (fieldName, componentType) => {
           ) {
             return false;
           }
-          const upperCasedFieldName = fieldName.charAt(0).toUpperCase() + fieldName.substring(1);
-          const attrName = `enable${upperCasedFieldName}Input`;
-          const isFixedValue = data?.openForms[fieldName]?.mode === 'fixedValue';
-          return isFixedValue && !data[attrName];
```

Comment:
@SilviaAmAm you seem to have touched this most recently, I grepped the codebase for `enable.*input` (case insensitive) and couldn't find any uses and it seems to work without. Do you remember if/why this was needed? From the git history it seemed like a leftover from my experiments/prototyping that should've been cleaned up before.

Reply:
No, i don't remember why this was needed :S

Reply:
good riddance then :)